backend/urls.py

```diff
@@ -22,8 +22,7 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('user/', views.UserView.as_view()),
-    path('historia/', views.HistoriaView.as_view()),
+    path('historia/', views.AppView.as_view()),
```

Comment:
Shouldn't we change the route from 'historia' to another thing?

Reply:
We decided together to keep 'historia' here for while.